### ISSUE TITLE
Streams won't play

### ISSUE THREAD

**Issue by Durhammer3111** [quodlibet/quodlibet#3518]
Steps to reproduce
------------------

Tried to stream these two stations and they don't work. They're fine in both Cantata and VLC:

http://playerservices.streamtheworld.com/pls/WTEBFM.pls?DIST=TuneIn&TGT=TuneIn&maxServers=2&tdtok=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6ImZTeXA4In0.eyJpc3MiOiJ0aXNydiIsInN1YiI6IjIxMDY0IiwiaWF0IjoxNTYyNTE0NjkzLCJ0ZC1yZWciOmZhbHNlfQ.7XtNruHNeD9OrfOuRRcsyokUFRLRXGJ4FvFtf0Pm8Dg

http://stream.abacast.net/playlist/sunrise-wrmrfmmp3-ibc2.m3u?source=tunein&source=TuneIn

Expected Output
---------------

Music, talk, ads, etc. !!!

Actual Output
-------------

Popup box: Playback Error -- This appears to be a text file. decodebin cannot decode plain text files.

Test System
-----------

### Which version of Quod Libet?

4.3.0, flatpak

### Which operating system 

MX Linux 19.3

### If it's audio-related, what back-end?

gstreamer 1.14.4 (-1?)

Additional Information
----------------------

New to QL. If there are logs, tell me where to find 'em, enable 'em, etc. I'll be happy to provide, but it might be easy enough to reproduce.

**Comment by declension** [comment]
Thanks

I get two different errors - streamtheworld exactly as described (note: served as `Content-Type: audio/x-scpls; charset=UTF-8`), but abacast.net one (served as `Content-Type: audio/x-mpegurl`) just doesn't play at all (no messages)

> If there are logs, tell me where to find 'em, enable 'em, etc. 
Just run with `--debug`, for now (see also #3466 )

**Comment by Durhammer3111** [comment]
Yup, sorry, you're right, the "Modern Rock" station seems to be a NOP. Nothing even shows up in the log, from what I could tell!

Attaching the log. QL is a flatpak so should we expect the "canberra-gtk-module" to already be there or not? I also see failure to load i915 and i965 modules, but don't know what those mean. These messages are only from the WTEB stream ("text file") attempt. NOTHING registers at all when I d-click the other station to play it.

Ah, okay, just realized the previous failed station might still have been in the queue, so restarted QL and tried playing the second station. This time, I DID get a different popup message: Playback Error: No GStreamer element found to handle media format. Media format: text/uri-list decoder.  Second ("Modern Rock") log is now attached with new messages.

[QL_Modern_Rock.log](https://github.com/quodlibet/quodlibet/files/5660170/QL_Modern_Rock.log)


[QL.log](https://github.com/quodlibet/quodlibet/files/5660075/QL.log)
